setup.py

```diff
@@ -112,8 +112,14 @@
       version=VERSION,
       description=DESCRIPTION,
       scripts=scripts,
-      install_requires=['numpy', 'matplotlib', 'scipy', 'poppy', 'astropy'],
-      setup_requires=['numpy', 'astropy'],
+      install_requires=[
+          'numpy>=1.10.0',
+          'matplotlib>=1.5.3',
```

Comment:
Leaving off the final / patch version number makes sense. I can go ahead and make that change.

I've been conditioned to trust the automatic dependency resolver :) I understand that enough of our colleagues have been burned by buggy software releases that they exercise an abundance of caution in upgrading. Nevertheless, user support wise, it's much easier to help someone through an "unsupported version" issue than a "something breaking in a subtle way in an untested configuration" issue.